app/main/views.py

```diff
@@ -87,25 +85,26 @@ def update_service(service_id):
     db.session.add(service)
     db.session.commit()
 
-    return jsonify(services=jsonify_service(service)), 200
+    return "", http_status
 
 
 @main.route('/services/<service_id>', methods=['GET'])
 def get_service(service_id):
-    service = Service.query.filter(Service.id == service_id).first_or_404()
+    service = Service.query.filter(Service.service_id == service_id)\
+                           .first_or_404()
```

Comment:
Yea, I've not added any of that stuff in this pull request. I'm hoping we can hold off on that stuff until after launch.